gabbi/case.py

```diff
@@ -27,7 +27,9 @@
 import re
 import sys
 
+import wsgi_intercept
```

Comment:
Actually apparently I'm using that style quite a bit throughout so may as well stick with it.